### ISSUE TITLE
Documentation for Github Profile Integration

### ISSUE THREAD

**Issue by Madfarm** [tipsy/profile-summary-for-github#126]
I think us noobies would appreciate some guidance on getting the data from this on our profiles

**Comment by Madfarm** [comment]
Yeah that's actually how I found this repo. Someone had the data displayed on their Github profile and it had a link to here. I'll go and try to find that profile for you and leave another comment when I do.

**Comment by Madfarm** [comment]
https://github.com/mithi

I originally thought they were just static images, but then I saw that it was updated with their commits from 2 days ago. They might just be updating the images as they work, but I'm not far enough in my development journey to be able to tell.

**Comment by iProdigy** [comment]
They're using https://github.com/vn7n24fzkq/github-profile-summary-cards

**Comment by tipsy** [comment]
Neat 😁
